Realm.XamarinAndroid/packages.config

```diff
@@ -1,4 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="DotNetCross.Memory.Unsafe" version="0.2.2" targetFramework="monoandroid60" />
+  <package id="Remotion.Linq" version="2.1.1" targetFramework="MonoAndroid60" />
```

Comment:
I think that's due to the value of the `TargetFrameworkVersion` property. In `Realm.XamarinAndroid` it's set to Latest, whereas in `IntegrationTests.XamarinAndroid` it's set to an older version.